src/extensions/default/bramble/lib/UI.js

```diff
@@ -249,13 +245,6 @@ define(function (require, exports, module) {
             return;
         }
 
-        // Switch the icon
-        $("#mobileViewButton").removeClass("desktopButton");
```

Comment:
This was code we did at the very, very beginning at CDOT, when we wanted to allow you to do various things from the status bar of the editor, before we had any UI in Thimble to control things.  It's dead now.